README.md

```diff
@@ -26,13 +26,15 @@
 
 **Goal:** Confirm proposal feasibility with Astro Maintainers and the [Technical Steering Committee (TSC)](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#technical-steering-committee-tsc).
 
-**Requirements:** An existing proposal (Stage 1). In addition, a proposal is more likely to be accepted if it is detailed and well thought-out, can demonstrate community interest, has at least one champion volunteer, and has buy-in/interest from Astro maintainer(s).
+**Requirements:** An existing proposal (Stage 1). In addition, a proposal is more likely to be accepted if it is detailed and well thought-out, can demonstrate community interest, has at least one champion volunteer, has a maintainer acting as a champion, and has buy-in/interest from Astro maintainer(s).
 
 **Location:** GitHub Issues [(see all accepted proposals).](https://github.com/withastro/roadmap/issues)
 
 **What to Expect:** A proposal reaches this stage (aka "is accepted") during a meeting with Maintainers and TSC, following our existing [RFC Proposal](https://github.com/withastro/.github/blob/main/GOVERNANCE.md#voting-rfc-proposals) voting process.
 
-When a proposal is accepted, a TSC member will create a new GitHub Issue summarizing the original proposal using our official template. At this point, the proposal champion is free to move on to the next stage. If a champion doesn't exist yet, then an accepted proposal may remain open until a champion volunteers by posting in the GitHub Issue.
+When a proposal is accepted, a TSC member will create a new GitHub Issue summarizing the original proposal using our official template.
+
+The proposal's original author will be given a chance to volunteer to be its champion. If they refuse, someone else can volunteer for the position by posting in the GitHub Issue. An accepted proposal will remain an open issue until a champion is confirmed by TSC. Once confirmed, the champion is free to advance the proposal to the next stage.
```

Comment:
I would be more precise and say that a champion "is confirmed" when the Stage 2 template is updated to reflect this.

Reply:
Currently a champion is not required to reach stage 2 and the text says this. Are you suggesting that we change it so that there is one? 

I could see wanting to do that, but the downside is some times a champion can't work on something right away, for example @natemoo-re proposal for the Container API. Maybe that was a past mistake we can avoid in the future though.

Reply:
Oh, I see now; I misunderstood things. However, this part is confusing: 

> Once confirmed, the champion is free to advance the proposal to the next stage.

Does that mean that Stage 2 proposals move forward only when they have a champion, meaning that the TSC doesn't need to be involved anymore?

Reply:
A proposal can't move to stage 3 (RFC PR) without a champion, yes. But it can move to stage 2. TSC is still involved approving moving to the stages beyond that.